video_worker/__init__.py

```diff
@@ -126,36 +112,31 @@ def run(self):
             )
 
         self.VideoObject.activate()
-        if self.VideoObject.valid is False:
-            ErrorObject().print_error(
-                message='Invalid Video / VEDA Data'
-            )
-            return None
+        if not self.VideoObject.valid:
+            logger.error('Invalid Video / VEDA Data')
+            return
 
         if not os.path.exists(self.workdir):
             os.mkdir(self.workdir)
 
-        """
-        Pipeline Steps :
-          I. Intake
-            Ib. Validate Mezz
-          II. change status in APIs
-          III. Generate Encode Command
-          IV. Execute Encodes
-            IVa. Validate Products
-          (*)V. Deliver Encodes (sftp and others?), retrieve URLs
-          (*)VI. Change Status in APIs, add URLs
-          VII. Clean Directory
+        # Pipeline Steps :
+        #   I. Intake
+        #     Ib. Validate Mezz
```

Comment:
It's short for "mezzanine", it's a video term for an intermediate file (or source file in this case).